Dockerfile

```
@@ -0,0 +1,21 @@
+FROM php:7.2-fpm
+
+# Install PHP extensions
+RUN docker-php-ext-install pdo_mysql && docker-php-ext-install calendar
+
+# Install Composer
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```

```suggestion
SHELL ["/bin/bash", "-o", "pipefail", "-c"]
RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```
You want to change the shell options to fail if anything in the pipe fails. Otherwise the image could build without having internet access and be totally broken.

---

Another great point, will change that.